code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -57,17 +56,21 @@
 		/datum/reagent/stable_plasma,
 		/datum/reagent/consumable/sugar,
 		/datum/reagent/sulfur,
-		/datum/reagent/toxin/acid,
+		/datum/reagent/toxin/acid
+	)
+	var/list/t2_upgrade_reagents = list(
+		/datum/reagent/iron,
 		/datum/reagent/water,
```

Comment:
```suggestion
```

Reply:
Remove water, why do we need water tanks if we have this

Reply:
> Remove water please, why do we need water tanks if we have this

I present to you. Sinks.

Reply:
And sinks

Reply:
Why do we need welder fuel if we have welder tanks?

I included the removed reagents in T2 for two reasons:
- People are gonna bitch, so this is a compromise
- It was the easiest route to fill an empty tier